### PR TITLE
refactor test for EmptyOrder component

### PR DESCRIPTION
**Pull request by Kharko7**
## Description

Refactor test for EmptyOrder  component by RTL.

#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| test before | after 2|

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/tests/unit/containers/orders/order/empty_order/tests/empty-order.test.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+}));
+
+jest.mock('@material-ui/styles', () => ({
+  ...jest.requireActual('@material-ui/styles'),
+  useTheme: () => ({
+    palette: {
+      type: 'light'
+    }
+  })
+}));
```
[the remark this answers is not in the document]
Comment (Kharko7) [reply src/tests/unit/containers/orders/order/empty_order/tests/empty-order.test.js]: Yes, changed

File: src/tests/unit/containers/orders/order/empty_order/tests/empty-order.test.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import { theme } from '../../../../../../../components/app/app-theme/app.theme';
+
+const themeValue = theme('light');
+
+describe('Empty order component tests', () => {
+  const buttonTitle = 'до каталогу';
+  const emptyTitle = 'Ваш кошик порожній';
+
+  it('should render title', () => {
+    const { container } = render(
```
Comment (tarasSanotskyi) [inline src/tests/unit/containers/orders/order/empty_order/tests/empty-order.test.js:15]: you don't need to write a render in every test case
use beforeEach

Comment (Kharko7) [reply src/tests/unit/containers/orders/order/empty_order/tests/empty-order.test.js]: Fixed, thanks

**Review by maarekua — DISMISSED** [review]

**Review by tarasSanotskyi — DISMISSED** [review]

**Review by maarekua — DISMISSED** [review]

File: src/tests/unit/containers/orders/order/empty_order/tests/empty-order.test.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```
Comment (rostyslaw27) [inline src/tests/unit/containers/orders/order/empty_order/tests/empty-order.test.js:1]: tests file name should have ending .spec.js

**Review by rostyslaw27 — CHANGES_REQUESTED** [review]

File: src/tests/unit/containers/orders/order/empty_order/tests/empty-order.test.js (hunk 3 above)
Comment (Kharko7) [reply src/tests/unit/containers/orders/order/empty_order/tests/empty-order.test.js]: Changed

File: src/tests/unit/containers/orders/order/empty_order/tests/empty-order.spec.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+          <EmptyOrder emptyTitle={emptyTitle} buttonTitle={buttonTitle} />
+        </Router>
+      </ThemeProvider>
+    );
+  });
+
+  it('should render title', () => {
+    const findeElement = screen.getByTestId('title');
+
+    expect(findeElement.textContent).toBe('Ваш кошик порожній');
```
Comment (rostyslaw27) [inline src/tests/unit/containers/orders/order/empty_order/tests/empty-order.spec.js:27]: Maybe it is better to replace 'Ваш кошик порожній' with emptyTitle variable

File: src/tests/unit/containers/orders/order/empty_order/tests/empty-order.spec.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  });
+
+  it('render text in button', () => {
+    const linkItem = screen.getByText('до каталогу');
```
Comment (rostyslaw27) [inline src/tests/unit/containers/orders/order/empty_order/tests/empty-order.spec.js:31]: Same here

File: src/tests/unit/containers/orders/order/empty_order/tests/empty-order.spec.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```
Comment (rostyslaw27) [inline src/tests/unit/containers/orders/order/empty_order/tests/empty-order.spec.js:1]: There is no need in folder "tests" in empty_order cause we are already in src/tests

**Review by rostyslaw27 — DISMISSED** [review]

File: src/tests/unit/containers/orders/order/empty_order/tests/empty-order.spec.js (hunk 4 above)
Comment (Kharko7) [reply src/tests/unit/containers/orders/order/empty_order/tests/empty-order.spec.js]: Changed

File: src/tests/unit/containers/orders/order/empty_order/tests/empty-order.spec.js (hunk 5 above)
Comment (Kharko7) [reply src/tests/unit/containers/orders/order/empty_order/tests/empty-order.spec.js]: Changed

File: src/tests/unit/containers/orders/order/empty_order/tests/empty-order.spec.js (hunk 6 above)
Comment (Kharko7) [reply src/tests/unit/containers/orders/order/empty_order/tests/empty-order.spec.js]: Changed

**Review by rostyslaw27 — APPROVED** [review]

**Review by LubomyrKashuba — APPROVED** [review]

**Pull request merged**